src/ValueObject/EncryptedColumn.php

```diff
@@ -43,4 +63,25 @@ public function getData()
     {
         return $this->data;
     }
+
+    /**
+     * @return string
+     */
+    public function getEncryptor(): string
```

Comment:
> Difficult to enable compatibility with other service containers

Just another method call there ;-)
Could you clarify on where the difficulty was?

 > (The string also allowed using it as a default value in the VO constructor, preserving BC) Thoughts?

Didn't understand this part